dags/nocp/scripts/run_ocm_benchmark.sh

```diff
@@ -59,7 +59,7 @@ run_ocm_benchmark(){
 
     start_time=\$(date +%s)
     echo "Running the ocm-load-test test"
-    build/ocm-load-test --config-file config.yaml &> ${UUID}_ocm-load-test
+    build/ocm-load-test --config-file config.yaml --log-file ${UUID}_ocm-load-test
     cp ${UUID}_ocm-load-test /tmp/${UUID}/results/${UUID}_ocm-load-test.json
```

Comment:
@venkataanil Can you explain why we are copying the log file to a new location and setting it as a .json? I couldn't find any reference to it after the copy and it isn't necessarily in json format. Thanks!

Reply:
our intention here is to move ${UUID}_ocm-load-test log file to snappy server. I think automation.py doesn't pick it up if it is not a json file. I exactly don't know where this check is (might be at https://github.com/cloud-bulldozer/ocm-api-load/blob/main/automation.py#L485 ), however unable to push earlier when json extension is not added.